src/quest_manager/templates/quest_manager/submission.html

```
@@ -79,7 +81,34 @@ <h3 class="panel-title">Quest Submission Form</h3>
-        {{ submission_form | crispy }}
+        {% if question_formset %}
+          {{ question_formset.management_form }}
+          {% if question_formset.non_form_errors %}
+            <div class="alert alert-danger">{{ question_formset.non_form_errors }}</div>
+          {% endif %}
+          {# the loop variable must be named `form`: the crispy HTML() layout objects inside #}
+          {# QuestionSubmissionForm reference {{ form.question... }} from the template context #}
+          {% for form in question_formset %}
+            <div class="well well-sm question-submission-form">
+              {# question number and the teacher's instructions on one line to save space; unwrap_p strips #}
+              {# summernote's wrapping <p> so a one-line prompt flows right after the "Question N:" label #}
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </p> ], start tag match failed [ <p> ] on line 94.

(tag-pair)